arches_lingo/src/arches_lingo/components/concept/ConceptImages/components/ConceptImagesViewer.vue

```diff
@@ -0,0 +1,195 @@
+import NonLocalizedStringWidget from "@/arches_component_lab/widgets/NonLocalizedStringWidget/NonLocalizedStringWidget.vue";
+
+const props = defineProps<{
+    tileData: ConceptImages | undefined;
+    componentName: string;
+    sectionTitle: string;
+    graphSlug: string;
+    nodegroupAlias: string;
+}>();
+
+const resources = ref<DigitalObjectInstance[]>();
+const { $gettext } = useGettext();
+const confirm = useConfirm();
+
+onMounted(async () => {
+    if (props.tileData) {
+        resources.value = await Promise.all(
+            props.tileData.aliased_data.depicting_digital_asset_internal.map(
+                async (digitalAsset) =>
+                    await getConceptImageResource(digitalAsset.resourceId),
+            ),
+        );
+    }
+});
+function confirmDelete() {
+    confirm.require({
+        header: $gettext("Confirmation"),
+        message: $gettext("Do you want to delete this concept image?"),
+        accept: () => {
+            console.log("do delete");
+        },
+        rejectProps: {
+            label: $gettext("Cancel"),
+            severity: SECONDARY,
+            outlined: true,
+        },
+        acceptProps: {
+            label: $gettext("Delete"),
+            severity: DANGER,
+        },
+    });
+}
+
+const openEditor = inject<(componentName: string) => void>("openEditor");
+async function getConceptImageResource(resourceInstanceId: string) {
+    try {
+        return await fetchLingoResource(
+            "digital_object_rdm_system",
+            resourceInstanceId as string,
+        );
+    } catch (error) {
+        console.error(error);
+    }
+}
+</script>
+
+<template>
+    <div class="section-header">
+        <h2>{{ props.sectionTitle }}</h2>
+        <Button
+            :label="$gettext('Add New Concept Image')"
+            @click="openEditor!(props.componentName)"
+        ></Button>
+    </div>
+    <div class="conceptImages">
+        <div
+            v-for="resource in resources"
+            :key="resource.resourceinstanceid"
+            class="conceptImage"
+        >
+            <div class="header">
+                <label
+                    for="conceptImage"
+                    class="text"
+                >
+                    <NonLocalizedStringWidget
+                        node-alias="name_content"
+                        graph-slug="digital_object_rdm_system"
+                        :mode="VIEW"
+                        :initial-value="
+                            resource.aliased_data.name.aliased_data.name_content
+                        "
+                    />
+                </label>
+                <div class="buttons">
+                    <Button
+                        icon="pi pi-file-edit"
+                        @click="openEditor!(props.componentName)"
+                    />
+                    <Button
+                        icon="pi pi-trash"
+                        :aria-label="$gettext('delete')"
```

Comment:
nit: `Delete`?

Reply:
Yes.  Mayhaps capitalization rules in the style guide?  Though this one does seem more like a typo.